### ISSUE TITLE
execute bash command over ssh

### ISSUE THREAD

**Issue by giovanidiniz** [coder/coder#10364]
With Coder v1 there was a CLI option to run an bash command over ssh. 
That option doesn't seem to be available in v2, so I wondered if there is any alternative to running a command directly onto the workspace from the CLI.

**Comment by sreya** [comment]
Hi @giovanidiniz you should be able to use `ssh` like you do with any other remote instance. E.g.

```
$ jon@aman ~ coder config-ssh
$ jon@aman ~ ssh coder.dev touch /home/coder/bar
$ coder@dev ~ ls /home/coder/bar
/home/coder/bar
``

**Comment by giovanidiniz** [comment]
Hi, Jon,

Thanks for the reply.

I was wondering if there would be a programmatic way of executing a command
using he CLI alone.

I have a python wrapper around the CLI but wondered if I could simplify.

I appreciate the insight

best

**Comment by mafredri** [comment]
@giovanidiniz sorry you ran into this. It's basic functionality that we should definitely support. I opened an issue in the past, but unfortunately it got closed by stale bot: #2154.

I'll close this issue and re-open that one, feel free to comment on #2154 if you have more to add to it. 👍🏻
